options.html

```diff
@@ -1,162 +1,129 @@
-        <label for="popupcolor">Pop-up color:</label>
-        <select id="popupcolor" name="popupcolor">
-            <option value="yellow">Yellow</option>
-            <option value="blue">Blue</option>
-            <option value="lightblue">Light Blue</option>
-            <option value="black">Black</option>
-        </select>
-    </p>
-
-    <hr>
-
-    <p>
-        The <b>Skritter Domain</b> setting allows you to change the URL that is
-        used for adding words to your Skritter queue (Skritter login required).
-        By default, <code>skritter.com</code> is being used, but for some users
-        <code>skritter.cn</code> might work better.
-    </p>
-    <p>
-        <label for="skritterTLD">Skritter domain:</label>
-        <select id="skritterTLD" name="skritterTLD">
-            <option value="com">skritter.com</option>
-            <option value="cn">skritter.cn</option>
-        </select>
-    </p>
-
-    <hr>
-
-    <span class="important">Don't forget to save your changes by clicking on the botton at the top of this page.</span>
-</form>
-
+    <div class="container">
+        <h1 class="my-3">Options for the Zhongwen Browser Extension</h1>
+
+        <h2 class="mb-3">Appearance of the Pop-up Window</h2>
+
+        <div class="form-group mb-4">
+            <label>
+                Background color of the pop-up window.
+                <small class="form-text text-muted">Changing this setting requires reloading the page with the pop-up to take effect.</small>
+            </label>
+            <div class="custom-control custom-radio">
+                <input type="radio" id="popupColorYellow" name="popupColor" class="custom-control-input">
+                <label class="custom-control-label" for="popupColorYellow">Yellow pop-up background</label>
+            </div>
+            <div class="custom-control custom-radio">
+                <input type="radio" id="popupColorBlue" name="popupColor" class="custom-control-input">
+                <label class="custom-control-label" for="popupColorBlue">Blue pop-up background</label>
+            </div>
+            <div class="custom-control custom-radio">
+                <input type="radio" id="popupColorLightblue" name="popupColor" class="custom-control-input">
+                <label class="custom-control-label" for="popupColorLightblue">Light blue pop-up background</label>
+            </div>
+            <div class="custom-control custom-radio">
+                <input type="radio" id="popupColorBlack" name="popupColor" class="custom-control-input">
+                <label class="custom-control-label" for="popupColorBlack">Black pop-up background</label>
+            </div>
+        </div>
+
+        <div class="form-group mb-4">
+            <label>
+                Coloring pinyin syllables based on the tone of the character.
+                <small class="form-text text-muted">Changing this setting requires reloading the page with the pop-up to take effect.</small>
```

Comment:
Strictly speaking, choosing not to show any tone colors doesn't require reloading the page, it is effective immediately. But we'll leave it like this, otherwise it's too confusing. :)